docs/admin/auth/methods.rst

```diff
@@ -156,11 +156,6 @@ It's recommended to have the ``alg`` (`Algorithm parameter`_)  in the header.
 If it's provided both in the token and in the response from the JWK endpoint,
 both values are compared and in case of a mismatch the token is rejected.
 
-JWT authentication requires retrieving public keys from a `JWK endpoint`_.
-CrateDB will cache these public keys for 10 hours, or according to the
-``Cache-Control: max-age`` value in the response header from the `JWK endpoint`_,
-if provided. This value must be specified in seconds as a positive integer;
-otherwise, it will be ignored.
```

Comment:
Given that we don't have default caching I think we must keep information (needs adjustment 10 hours is irrelevant).

Users can forget about adding that header and it will lead to performance decrease as originally reported

Reply:
I don't see why. "The implementation uses Cache-Control headers to cache result" is kind of just doing HTTP right?

Reply:
Okay fair point - added a fixup